packages/framework/ini-files/config-specs.ini

```diff
@@ -1815,6 +1899,54 @@ opt-set-cmake-var Trilinos_ENABLE_TrilinosBuildStats     BOOL FORCE : OFF
 
 use RHEL8_POST
 
+[rhel8_aue-gcc-openmpi_release_static_no-kokkos-arch_no-asan_complex_no-fpic_mpi_no-pt_no-rdc_no-uvm_deprecated-on_compsim]
+use COMPILER|GNU
+use BUILD-TYPE|RELEASE
+use NODE-TYPE|SERIAL
+# Shared libraries until static ones are available in AUE container
+opt-set-cmake-var BUILD_SHARED_LIBS      BOOL   : ON
```

Comment:
Can you set `BUILD_SHARED_LIBS` to `OFF` but leave `TPL_FIND_SHARED_LIBS` to `ON`?